### PR TITLE
Contact list pagination

### LINKED ISSUE #57: Contacts pagination
Implement pagination to contact list

### PR DESCRIPTION
**Pull request by icerrate**
Close #57

### CODE REVIEW

File: app/src/main/java/com/belatrixsf/allstars/utils/Constants.java [hunk 1]
```diff
@@ -14,4 +14,6 @@
     public static final int FIRST_POSITION = 1;
     public static final int SECOND_POSITION = 2;
     public static final int THIRD_POSITION = 3;
+
+    public static final String EMPTY_STRING = "";
```
Comment (gyosida) [inline app/src/main/java/com/belatrixsf/allstars/utils/Constants.java]: omg

File: app/src/main/java/com/belatrixsf/allstars/ui/contacts/ContactsListPresenter.java [hunk 2]
```diff
@@ -38,9 +40,12 @@
 public class ContactsListPresenter extends AllStarsPresenter<ContactsListView> {
 
     private EmployeeService employeeService;
-    private List<Employee> employees;
+    private List<Employee> contacts = new ArrayList<>();
     private boolean inActionMode = false;
     private boolean profileEnabled = true;
+    private PaginatedResponse contactPaginatedResponse = new PaginatedResponse();
+    private int currentPage = Constants.FIRST_POSITION;
```
Comment (gyosida) [inline app/src/main/java/com/belatrixsf/allstars/ui/contacts/ContactsListPresenter.java]: why do we need an APPLICATION constant for a "first position" value?

File: app/src/main/java/com/belatrixsf/allstars/ui/contacts/ContactsListFragment.java [hunk 3]
```diff
@@ -216,7 +256,7 @@ public boolean onCreateActionMode(final ActionMode mode, Menu menu) {
             cleanImageButton.setOnClickListener(new View.OnClickListener() {
                 @Override
                 public void onClick(View v) {
-                    searchTermEditText.setText("");
+                    searchTermEditText.setText(Constants.EMPTY_STRING);
```
Comment (gyosida) [inline app/src/main/java/com/belatrixsf/allstars/ui/contacts/ContactsListFragment.java]: ?

**Comment by icerrate** [comment]
Close #79

**Pull request merged**